src/app/shared/components/filters-list/searchbar/searchbar.component.ts

```diff
@@ -96,6 +96,13 @@ export class SearchbarComponent implements OnInit, OnDestroy {
     }
   }
 
+  public onDeletePreviousSearchValue(value: string, event: Event): void {
+    event.stopPropagation();
+    this.previousResults = this.previousResults.filter((result: string) => result !== value);
+    localStorage.setItem('previousResults', JSON.stringify(this.previousResults));
```

Comment:
[optional].
I like to have everything under control in one place, so it's better to move it to the NGXS store (we already store part of the store in the browser's SessionStorage). 
We can leave it for now, and in the future we can finalize

Reply:
It is need to check moving functionality to NGXS store.